react-scanner.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+	// Crawl the entire repo
+	crawlFrom: './',
+	// Needed for properly reporting components with dot notation
+	includeSubComponents: true,
+	// Exclude usage in tests and stories.
+	globs: [ '**/!(test|stories)/!(*stories).@(js|ts)?(x)' ],
+	// Exclude any vendor or docs directories
+	exclude: [
+		'bin',
+		'build',
+		'build-tools',
+		'config',
+		'docs',
+		'node_modules',
+		'public',
+		'results',
+		'static',
+		'test',
+	],
+	// Consider only imports of `@wordpress/components`
+	importedFrom: '@wordpress/components',
+	// Full usage report
+	processors: [ [ 'raw-report', { outputTo: './results/calypso.json' } ] ],
```

Comment:
I understand this is mostly just for automated execution on a server somewhere, but it might be helpful to have `components-usage` or `components-usage-stats` in the folder name, instead of just `results`.

Reply:
I agree - I got this feedback multiple times and I'll be addressing it, first in Gutenberg and in the related script, then here and in the Jetpack PR.